src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/InvitationAggregate/InvitationTests.cs

```diff
@@ -821,6 +821,49 @@ public void EditIpo_ShouldAddEditIpoEvent()
         }
         #endregion
 
+        #region UpdateAttendedStatus
+        [TestMethod]
+        public void UpdateAttendedStatus_ShouldNotUpdateAttendedStatus_WhenIpoIsCanceled()
+            => Assert.ThrowsException<Exception>(()
+                => _dutWithCanceledStatus.UpdateAttendedStatus(
+                    _functionalRoleParticipant,
+                    true,
+                    _functionalRoleParticipant.RowVersion.ConvertToString()));
+
+        [TestMethod]
+        public void UpdateAttendedStatus_ShouldUpdateAttendedStatusEvent()
+        {
+            _dutDpIpo.UpdateAttendedStatus(
+                _personParticipant,
+                true,
+                _personParticipant.RowVersion.ConvertToString());
+
+            Assert.IsInstanceOfType(_dutDpIpo.PreSaveDomainEvents.Last(), typeof(AttendedStatusUpdatedEvent));
+        }
+        #endregion
+
+
+        #region UpdateNote
+        [TestMethod]
+        public void UpdateNote_ShouldNotUpdateNote_WhenIpoIsCanceled()
+            => Assert.ThrowsException<Exception>(()
+                => _dutWithCanceledStatus.UpdateNote(
+                    _functionalRoleParticipant,
+                    "note",
+                    _functionalRoleParticipant.RowVersion.ConvertToString()));
+
+        [TestMethod]
+        public void UpdateNote_ShouldUpdateNoteEvent()
```

Comment:
testname mismatch what it tests .. This test test that the event is added. Duplicate this test, and also Assert that the note is changed